docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,251 @@
+You can add `--dry-run` option to create the submissions files first and check them, and then submit the jobs with `condor_submit condor_scans_zh.sub`. 
+
+After all jobs are completed we can combine the files for each POI: 
+
+```shell
+for p in r_zh_75_150 r_zh_150_250noj r_zh_150_250wj r_zh_250_400 r_zh_gt400
+do
+    hadd -k -f scan_${p}_blinded.root higgsCombine.scans_blinded.${p}.POINTS.*.MultiDimFit.mH120.root
+done
+```
+
+And finally plot the likelihood scans 
+
+```shell
+python scripts/plot1DScan.py scan_r_zh_75_150_blinded.root  -o scan_r_zh_75_150_blinded --POI r_zh_75_150 --json summary_zh_stxs_blinded.json
+```
+![](figures/scan_r_zh_75_150_blinded.png)
+
+## Impacts
+
+One of the important tests before we move to the unblinding stage is to check the impacts of nuisance parameters on each POI. For this we can run the `combineTool.py` with `-M Impacts` method. We start with the initial fit, which should take ~20 minutes (good time to have a coffee break!):
+ 
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --robustFit 1 --doInitialFit --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400
+```
+
+Note that it is important to add the option `--redefineSignalPOIs [list of parameters]`, to produce the impacts for all POIs we defined when the workspace was created with the `multiSignalModel`.
+
+After the initial fit is completed we can perform the likelihood scans for each nuisance parameter as shown below
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --robustFit 1 --doFits --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --job-mode condor --task-name impacts_zh 
+```
+We will submit the jobs to the HTCondor to speed up the process.
+
+Now we can combine the results into the `.json` format and use it to produce the impact plots.
+
+```shell
+combineTool.py -M Impacts -d ws_full.root -m 125 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --output impacts.json 
+
+plotImpacts.py -i impacts.json -o impacts_r_zh_75_150 --POI r_zh_75_150
+```
+
+![](figures/impacts.png)
+> Do you observe differences in impacts plots for different POIs, do these differences make sense to you? 
+
+
+## Unfolded measurements
+
+Now that we studied the NP impacts for each POI, we can finally extract the measurements.
+Note, that in this exercise we are skipping couple of checks that have to be done before the unblinding. Namely the goodness of fit test and the post-fit plots of folded observables. Both of these checks were detailed in the previous exercises, you can find the description under the following links. 
```

Comment:
"skipping couple of checks" --> Maybe something like "we are skipping further checks and validation that you should do on your analysis for the purposes of the tutorial."